rekcurd_dashboard/apis/api_evaluation.py

```diff
@@ -195,6 +195,7 @@ def post(self, project_id: int, application_id: str):
         args = self.eval_parser.parse_args()
         eval_id = args.get('evaluation_id', None)
         model_id = args.get('model_id')
+        do_overwrite = args.get('overwrite')
```

Comment:
At the moment, it is OK.
Hopefully, we should use `PATCH` or `PUT` method for this.

Reply:
oh, that's right. I will fix it

Reply:
@keigohtr I removed `overwrite` parameter and use `PUT` to re-evalualte model
backend: https://github.com/rekcurd/dashboard/pull/105/commits/0fb0dd67c39be7ccc0996c1aa50c761fa84940db
frontend: https://github.com/rekcurd/dashboard/pull/105/commits/fe95fcb06eb3e3cbb78c83f200115382ff2d3595